nginx/metakgp.org

```diff
@@ -41,10 +41,6 @@ server {
 		}
 	}
 
-	location /peqp {
-		try_files $uri $uri/ =404;
-	}
-
 	location ~ ^/google557cb96b33ddc6b5\.html$ {}
 	
 	location /images {
```

Comment:
For the mediawiki volume to be truly ephemeral, we also want to put image uploads into the static volume. You could take a look at whether it's possible to configure media uploads to a separate subdomain, but I think the simplest solution might be to symlink the images dir to a dir in the static volume. Feel free to defer this to a later patch though.

Reply:
Yeah, I got it. Will defer to a later patch.